admin/osx/sign_app.sh

```diff
@@ -26,7 +26,7 @@ team_identifier="$3"
 app_domain="$4"
 
 codesign -s "$identity" --force --verbose=4 --options=runtime "$src_app/Contents/Frameworks/Sparkle.framework/Versions/B/XPCServices/Installer.xpc"
-codesign -s "$identity" --force --verbose=4 --options=runtime --entitlements "$HOME/Sparkle/Entitlements/Downloader.entitlements" "$src_app/Contents/Frameworks/Sparkle.framework/Versions/B/XPCServices/Downloader.xpc"
```

Comment:
This will continue to work independently of the version of Sparkle? I mean: should I upgrade my Sparkle installation?

Reply:
Actually, since we removed the use of entitlements, it should work anyway with older versions of Sparkle that contain these entitlements. But you should migrate to at least version 2.6.2 (https://github.com/sparkle-project/Sparkle/releases/tag/2.6.2) because it contains an important security fix